provision/provision.sh

```diff
@@ -476,9 +463,9 @@ if [[ $ping_result == "Connected" ]]; then
 		fi
 	fi
 	# Install the standards in PHPCS
-	/srv/www/phpcs/scripts/phpcs --config-set installed_paths ./CodeSniffer/Standards/WordPress/
-	/srv/www/phpcs/scripts/phpcs --config-set default_standard WordPress-Core
-	/srv/www/phpcs/scripts/phpcs -i
+	sudo /srv/www/phpcs/scripts/phpcs --config-set installed_paths /srv/www/phpcs-wordpress/
```

Comment:
This absolute path will break PHPCS when invoked from outside of Vagrant. This is why a relative path is used currently.